editor/editor_properties.cpp

```diff
@@ -328,12 +345,24 @@ EditorPropertyTextEnum::EditorPropertyTextEnum() {
 
 ///////////////////// PATH /////////////////////////
 
+void EditorPropertyPath::set_read_only(bool p_read_only) {
+	path->set_editable(!p_read_only);
+	path_edit->set_disabled(p_read_only);
+	EditorProperty::set_read_only(p_read_only);
+};
+
 void EditorPropertyPath::_path_selected(const String &p_path) {
+	if (is_read_only()) {
+		return;
+	}
 	emit_changed(get_edited_property(), p_path);
 	update_property();
 }
 
 void EditorPropertyPath::_path_pressed() {
+	if (is_read_only()) {
+		return;
+	}
```

Comment:
Will this ever be called when the property is disabled? The button is disabled, so it can't emit the signal connected to this method.

Reply:
I remember that `path_selected` was checked `is_read_only()` because the event fired even after the button was disabled. Rather, check why the event fires and I should probably fix that one.

Reply:
I didn't input the value for setup properly when testing, so it seems that another factor was causing the crash. So I think it's OK to omit it.